src/script/auth/page/ConversationJoin.tsx

```diff
@@ -60,13 +60,13 @@ const ConversationJoinComponent = ({
   doRegisterWireless,
   setLastEventDate,
   doLogout,
-  selfName,
+  self,
   conversationError,
 }: Props & ConnectedProps & DispatchProps) => {
   const nameInput = React.useRef<HTMLInputElement>(null);
   const {formatMessage: _} = useIntl();
+  const [isLoggedIn, setIsLoggedIn] = useState<boolean>();
```

Comment:
suggestion(non-blocking): Let's add false as initial value, this will make isLoggedIn type `boolean` instead of `boolean | undefined`.